geode-core/src/upgradeTest/java/org/apache/geode/internal/cache/rollingupgrade/RollingUpgradeDUnitTest.java

```diff
@@ -205,6 +210,22 @@ void doTestRollAll(String regionType, String objectType, String startingVersion)
       verifyValues(objectType, regionName, 0, 10, server2);
       putAndVerify(objectType, server2, regionName, 15, 25, server1);
 
+      final short versionOrdinalAfterUpgrade = Version.getCurrentVersion().ordinal();
+      locator.invoke(() -> {
+
+        final Locator theLocator = Locator.getLocator();
+        final DistributedSystem distributedSystem = theLocator.getDistributedSystem();
+        final InternalDistributedSystem ids =
+            (InternalDistributedSystem) distributedSystem;
+        final DistributionManager distributionManager = ids.getDistributionManager();
+        final MembershipView<InternalDistributedMember> view =
+            distributionManager.getDistribution().getView();
+
+        for (final InternalDistributedMember member : view.getMembers()) {
+          assertThat(member.getVersionOrdinal()).isEqualTo(versionOrdinalAfterUpgrade);
+        }
+      });
```

Comment:
This is the new rolling upgrade test logic that verifies that after the upgrade, all the `MemberIdentifiers` in the view, report the current version (not an old version.)

This is adapted from the original Ericsson PR that showed us the bug that became GEODE-8240. Rather than use gfsh, we are delving into the `IDM` directly here.